packages/web/config/chain-infos.ts

```diff
@@ -167,6 +167,16 @@ const chainInfos = (
           coinGeckoId: "alter",
           coinImageUrl: "/tokens/alter.svg",
         },
+        {
+          type: "cw20",
+          contractAddress: "secret1s09x2xvfd2lp2skgzm29w2xtena7s8fq98v852",
+          coinDenom: "ALTER",
```

Comment:
coinDenom must match "AMBER" and not "ALTER"

Reply:
fixed, thanks